api/kessel/inventory/v1beta1/resources/notifications_integrations_service.proto

```diff
@@ -38,10 +39,18 @@ message DeleteNotificationsIntegrationRequest {
 message DeleteNotificationsIntegrationResponse {}
 
 message ListNotificationsIntegrationsRequest {
+	// resource type and parent relationship
+	// permission (unless some variation of 'this is the permission used for visibility' can be baked into the schema- this is different from the permission parameter in CheckForView because only checks for actual visibility likely matter here)
```

Comment:
Interesting, I wonder if type -> list permission type mappings will be part of `correlation` (https://miro.com/app/board/uXjVKoN7moU=/?moveToWidget=3458764617439715949&cot=14).